src/ts/content/entities/bookForm/data.ts

```diff
@@ -10,6 +10,13 @@ const ROLES_INPUT_COUNT_KEY = "___roles-count_";
 
 // IDs of elements we don't want to copy/paste
 const ID_BLACKLIST = new Set(["item_inventory_barcode_1"]);
+const TRANSFORMERS = {
```

Comment:
I know what you're thinking:
could you implement ID blacklist with `() => ""`?

Well no. If you put in a barcode before pasting, the barcode should remain. That's the one thing that is unique to each form, and should never ever get modified I think.

Reply:
nvm figured it out